pydatastructs/linear_data_structures/Mutidimensional Array.py

```diff
@@ -0,0 +1,36 @@
+# API to create N dimensional Array
+'''
+Data Members
+1. _ndim - The number of dimensions in the array.
+2. _data - The python list which will store the data in row major order.
+3. _dtype - The data type of the elements.
+
+Methods
+1. __new__(cls, dtype=NoneType, *args, **kwargs)
+2. __getitem__(self, *indices)
+3. __setitem__(self, value, *indices)
```

Comment:
Can we have `__setitem__(self, *indices_value)`? `value` can be obtained from `value = indices_value[-1]`.

Reply:
Yes, okay